### PR TITLE
Add endpoint for updating customizations in batches

### PR DESCRIPTION
**Pull request by fabubaker**
Creates a new endpoint, `/batch-update-customization` for sending batches of customization updates to RayyanStore. 

Additionally, a `log_enabled` parameter has been added to customization modifying endpoints. 
`log_enabled` toggles whether the customization should be logged to the customizations log or not.
This was implemented by adding an additional parameter to the prepared statement 
 `r${reviewId}_update_customization`.

### CODE REVIEW

File: server/src/adapters/postgres/postgres.js [hunk 1]
```diff
@@ -735,7 +737,7 @@ module.exports = {
       DROP FUNCTION IF EXISTS r${reviewId}_insert_customization(INTEGER, INTEGER, TEXT, JSONB);
       DROP FUNCTION IF EXISTS r${reviewId}_delete_customization(INTEGER, INTEGER, TEXT);
       DROP FUNCTION IF EXISTS r${reviewId}_log_customization(BOOLEAN, INTEGER, INTEGER, TEXT, TEXT);
-      DROP FUNCTION IF EXISTS r${reviewId}_update_customization(BOOLEAN, INTEGER, INTEGER, TEXT, JSONB);
+      DROP FUNCTION IF EXISTS r${reviewId}_update_customization(BOOLEAN, INTEGER, INTEGER, TEXT, JSONB, BOOLEAN);
```
Comment (fabubaker) [inline server/src/adapters/postgres/postgres.js]: This change will require a migration.

Comment (hammady) [reply server/src/adapters/postgres/postgres.js]: ok

File: server/src/adapters/postgres/postgres.js [hunk 2]
```diff
@@ -1231,7 +1233,7 @@ module.exports = {
       })
   },
 
-  updateCustomization: function (insert, reviewId, articleId, userId, key, value) {
+  updateCustomization: function (insert, reviewId, articleId, userId, key, value, logEnabled = true) {
     // TODO support bulk actions (multiple articles/keys)
```
Comment (hammady) [inline server/src/adapters/postgres/postgres.js]: should we remove this comment now?

File: server/src/adapters/postgres/postgres.js (hunk 1 above)
Comment (hammady) [reply server/src/adapters/postgres/postgres.js]: Won't this cause deletions of old reviews not to drop the function as it won't match the signature?
Can we instead have 2 `DROP` statements?

Comment (fabubaker) [reply server/src/adapters/postgres/postgres.js]: Good catch, I'll add the 2nd `DROP`.

**Comment by fabubaker** [comment]
Related to this PR is the issue of importing the customizations_log through the `/import` endpoint. 

We need to update `r${reviewId}_customizations_log_id_seq`, which is the sequence used for ids in the customizations_log table, after `/import` finishes it's job. 

Otherwise, when the user performs customizations, there will be rows with the same `_id`. While functionally this does not cause any bugs (for now), we should update the sequence to be consistent.

**Pull request merged**